.nginx.conf

```diff
@@ -0,0 +1,66 @@
+##
+# Put this file in /etc/nginx/conf.d folder and make shure
+# you have line 'include /etc/nginx/conf.d/*.conf;'
+# in your main nginx configuration file
+##
+
+##
+# Redirect to the same URL with https://
+##
+
+server {
+
+  listen 80;
+
+# Type your domain name below
+  server_name example.com;
+
+  return 301 https://$server_name$request_uri;
+
+}
+
+##
+# HTTPS configurations
+##
+
+server {
+
+  listen 443;
+
+# Type your domain name below
+  server_name example.com;
```

Comment:
Instead of repeating the `server_name` variable, can we put it at the top so people only had to change it once?